pandas/tests/internals/test_api.py

```diff
@@ -27,10 +27,9 @@ def test_namespace():
     ]
     expected = [
         "Block",
-        "NumericBlock",
         "DatetimeTZBlock",
         "ExtensionBlock",
-        "ObjectBlock",
+        "NumpyBlock",
```

Comment:
id be OK with not even having this go in the semi-public API.  need to ween downstream off of this anyway.  cc @mroeschke